main.js

```diff
@@ -160,4 +160,249 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// **----------  OPDRACHT 1A ----------**
+
+const typeTvAndNames = inventory.map ((stockTv)=>{
+  return stockTv.type +" "+ stockTv.name;
+})
+ console.log(typeTvAndNames);
+
+
+// **----------  OPDRACHT 1B ----------**
+
+const soldOutTvs = inventory.filter ((stockTv)=>{
+  return (stockTv.originalStock - stockTv.sold) === 0;
```

Comment:
Je kan dit nog iets korter schrijven
```suggestion
  return stockTv.originalStock === stockTv.sold;
```